src/test/scala/org/neo4j/spark/Neo4jDataFrameScalaTSE.scala

```diff
@@ -44,9 +28,8 @@ class Neo4jDataFrameScalaTest {
     val df = new SQLContext(sc).createDataFrame(rows, schema)
     Neo4jDataFrame.mergeEdgeList(sc, df, ("Person",Seq("name")),("ACTED_IN",Seq.empty),("Movie",Seq("title")))
 
-    val it: ResourceIterator[Long] = server.graph().execute("MATCH (:Person {name:'Keanu'})-[:ACTED_IN]->(:Movie {title:'Matrix'}) RETURN count(*) as c").columnAs("c")
-    assertEquals(1L, it.next())
-    it.close()
+    val count = SparkConnectorScalaSuiteIT.session().run("MATCH (:Person {name:'Keanu'})-[:ACTED_IN]->(:Movie {title:'Matrix'}) RETURN count(*) as c").single().get("c").asLong()
```

Comment:
consider using triple-quotes for these long statements, but that can also happen in a later PR